src/api/test/functional/webui/spider_test.rb

```diff
@@ -7,6 +7,28 @@
 ENV['BOOTSTRAP'] = '1'
 
 class Webui::SpiderTest < Webui::IntegrationTest
+  def ignore_link?(link)
+    return 1 if link =~ %r{/mini-profiler-resources}
```

Comment:
nitpick: `return true...`

Reply:
Yeah, that happens if you mix perl and ruby between day and night :)

Reply:
fixed